version.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/sh
+
+# Copyright (c) 2018 Anton Semjonov
+# Licensed under the MIT License
```

Comment:
Just to confirm, is MIT GPL2 compatible?

cc @lsm5 @giuseppe

Reply:
Oh whoops, I didn't even think about that. It [appears so](https://www.gnu.org/licenses/license-list.html#X11License), however.

Reply:
Yes, MIT is GPLv2 compatible.